resolwe/flow/tests/test_commands.py

```diff
@@ -77,6 +77,34 @@ def test_inherit_perms(self):
         self.assertEqual(UserObjectPermission.objects.filter(user=self.user).count(), 2)
         self.assertTrue(self.user.has_perm('flow.view_process', process))
 
+    def test_retire(self):
+        # No process should be in data base initially
+        self.assertEqual(Process.objects.count(), 0)
+
+        out, err = StringIO(), StringIO()
+        call_command('register', stdout=out, stderr=err)
+
+        # Check that all registered processes are active
+        initial_processes = Process.objects.count()
+        active_processes = Process.objects.filter(is_active=True).count()
+        self.assertGreater(initial_processes, 0)
+        self.assertEqual(initial_processes, active_processes)
+
+        # Check that retired processes without data are removed and
```

Comment:
Add a test that `register` without `--retire` doesn't do this.

Reply:
done